pkg/kclient/operators.go

```diff
@@ -191,6 +220,23 @@ func (c *Client) GetRestMappingFromUnstructured(u unstructured.Unstructured) (*m
 	return mapper.RESTMapping(gvk.GroupKind(), gvk.Version)
 }
 
+func (c *Client) GetRestMappingFromGVK(gvk schema.GroupVersionKind) (*meta.RESTMapping, error) {
```

Comment:
You could factorize this function and GetRestMappingFromUnstructured which are very similar.

Reply:
I have added a TODO for it, but again, this PR is long as so I'll take it up in a separate PR.